sync/awsdatabricks.py

```diff
@@ -335,7 +336,11 @@ def _get_aws_cluster_info_from_s3(bucket: str, file_key: str, cluster_id):
         logger.warning(f"Failed to retrieve cluster info from S3 with key, '{file_key}': {err}")
 
 
-def monitor_cluster(cluster_id: str, polling_period: int = 20) -> None:
+def monitor_cluster(
+    cluster_id: str,
+    polling_period: int = 20,
+    cluster_report_destination_override: dict = None,
```

Comment:
is it better to type the None as Optional too?

```suggestion
    cluster_report_destination_override: Optional[dict] = None,
```